.github/workflows/deploy.yaml

```diff
@@ -17,3 +17,12 @@ jobs:
         env:
           GITHUB_MAVEN_USERNAME: ${{ secrets.GITHUB_MAVEN_USERNAME }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          MAVEN_RELEASES_URL: https://maven.pkg.github.com/atlassian/packages-maven
```

Comment:
We don't need `MAVEN_RELEASES_URL` anymore since umbrella-type repos do not works for our purpose